Makefile

```diff
@@ -499,3 +505,46 @@ ironic_deploy_cleanup: ## cleans up the service instance, Does not affect the op
 	$(eval $(call vars,$@,ironic))
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${OPERATOR_BASE_DIR}/ironic-operator ${DEPLOY_DIR}
+
+##@ NOVA
+.PHONY: nova_prep
+nova_prep: export IMAGE=${NOVA_IMG}
+nova_prep: ## creates the files to install the operator using olm
+	$(eval $(call vars,$@,nova))
+	bash scripts/gen-olm.sh
+
+.PHONY: nova
+nova: namespace nova_prep ## installs the operator, also runs the prep step. Set NOVA_IMG for custom image.
+	$(eval $(call vars,$@,nova))
+	oc apply -f ${OPERATOR_DIR}
+
+.PHONY: nova_cleanup
+nova_cleanup: ## deletes the operator, but does not cleanup the service resources
+	$(eval $(call vars,$@,nova))
+	bash scripts/operator-cleanup.sh
+	rm -Rf ${OPERATOR_DIR}
+
+.PHONY: nova_deploy_prep
+nova_deploy_prep: export KIND=Nova
+# TOOD(gibi): the tooling expect a containerImage at the top level
+# but for projects like Cinder and Nova where there are multiple services with
+# different images this customization does not make sense. Make this
+# customization optional in the tooling.
+nova_deploy_prep: export IMAGE="unused"
```

Comment:
this is the image of the operator
there will only be one operator image for nova so i think your got confused with the container images used by the nova binary contianers

Reply:
oh never mind this is the deploy one ya hova will have several